commands/build.go

```diff
@@ -38,6 +36,8 @@ func runBuildCommand(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	projectCtx.BuildDir = projectCtx.Path
```

Comment:
Could we specify it near `projectCtx.Path = cmd.Flags().Arg(0)`
Looks like:
```
var1 = ...
some_magic_with_var_1_maybe()
var2 = var1
```

Was `var1` changed or not? It's not clear.

Reply:
It's filled in `project.FillCtx` function. It should be obvious that it changes context.